### PR TITLE
change the signature of backfillFromConfig

### PR DESCRIPTION
**Pull request by piuccio**
@janua @rich-nguyen

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+#### 0.68
+
+  - Change the signature of `FAPI.backfillFromConfig` to `FAPI.backfillFromConfig(collectionConfig: CollectionConfig)`
```
Comment (janua) [inline CHANGES.md]: Add the parameter

**Comment by janua** [comment]
:+1:

File: CHANGES.md (hunk 1 above)
Comment (piuccio) [inline CHANGES.md]: like this?

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+#### 0.68
+
+  - Change the signature of `FAPI.backfillFromConfig` to `FAPI.backfillFromConfig(collectionConfig: CollectionConfig, adjustSearchQuery: AdjustSearchQuery, adjustItemQuery: AdjustItemQuery)`
```
Comment (janua) [inline CHANGES.md]: No, I meant what you are changing it from. Doesn't really matter too much! But definitely not the full signature :)

Comment (piuccio) [inline CHANGES.md]: oh like this then :grin:

**Comment by rich-nguyen** [comment]
nice :+1:

**Pull request merged**